portfolio/src/main/webapp/2020.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>sehhaj portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+        <h1>sehhaj's portfolio - still to be implementated</h1>
+    </div>
+  </body>
+
+  <div id="nav-bar">
```

Comment:
Something to think about for a later PR:
this nav bar is copied and pasted on every html page, generally speaking we try to avoid copying code so something you could do is pop the navbar in it's own html file and use some javascript to load it into the page when needed.

Reply:
Made a separate nav-bar.html that has the nav-bar code and call it into each html with javascript